test/requirements.txt

```diff
@@ -1,3 +1,5 @@
 addict
 modelscope
 gguf
+gptqmodel
```

Comment:
rm gptq model,  test only handle cpu test

Reply:
besides, as you port all the code and use torch to do the gemm, gptqmodel is not needed for 3bits